pkgs/by-name/th/thc-secure-delete/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "thc-secure-delete";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gordrs";
+    repo = "thc-secure-delete";
+    rev = finalAttrs.version;
+    hash = "sha256-hvWuxFkzhOSCplPtyjRtn36bIk6KdPBcpr3lAmiAyfE=";
+  };
+
+  build-systems = [ pkg-config ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
```

Comment:
Remove this, it's not needed.

Reply:
so exactly how am i going to output the binaries if i remove mkdir -p $out/bin?